napari/layers/utils/_text_utils.py

```diff
@@ -30,13 +30,32 @@ def _calculate_anchor_center(
 
 
 def _calculate_bbox_centers(view_data: Union[np.ndarray, list]) -> np.ndarray:
+    """
+    Calculate the bounding box of the given centers,
+
+    Parameters
+    ----------
+    view_data : np.ndarray | list of ndarray
+        if an ndarray, return the
```

Comment:
it looks like you may have forgotten to finish this docstring?

Reply:
oops, yes, thanks.